source/simulator/helper_functions.cc

```diff
@@ -909,8 +909,142 @@ namespace aspect
 
     return (material_model->get_model_dependence().viscosity != MaterialModel::NonlinearDependence::none);
   }
-}
 
+
+
+// apply limiter for dg solutions
```

Comment:
This comment is correct (you can extend it a bit if you want), but please move it into the doxygen format of the lines below (the content of the comment below is copy-pasted anyway), and move this comment into the header file (supposedly simulator.h, take a look where normalize_pressure is defined).